src/install-edge-runtime.sh

```diff
@@ -6,9 +6,45 @@
 #script to install edge runtime 1.2
 
 ######################################
-# install_edge_runtime
+# apply_config_changes
 #
-#    - installs Azure IoT Edge Runtime 1.2
+#    - apply changes and restart
+#
+# ARGUMENTS:
+# OUTPUTS:
+#    Write output to stdout
+# RETURN:
+######################################
+
+function apply_config_changes() {
+    log_info "Apply settings"
+    iotedge config apply 2>>$STDERR_REDIRECT 1>>$STDOUT_REDIRECT
+    exit_code=$?
+    if [[ $exit_code == 0 ]];
+    then
+        log_info "IotEdge has been configured successfully"
+    fi
+
+    log_info "Restart Azure IoTEdge"
+    iotedge system restart 2>>$STDERR_REDIRECT 1>>$STDOUT_REDIRECT
+    exit_code=$?
+    if [[ $exit_code == 0 ]];
+    then
+        log_info "IotEdge has been restarted successfully"
+    fi
+
+    for next_symbol in '-' '\\' '|' '/';
+    do
+        echo -en "$next_symbol\b"
+        sleep 0.35
+    done
+    echo -en " \b"
```

Comment:
what does next_symbol represent here? Is this sleep to allow the Edge restart to complete?

Reply:
I was trying something - not needed!